sunpy/coordinates/tests/test_metaframes.py

```diff
@@ -88,7 +88,7 @@ def test_class_creation(indirect_fixture):
     assert base_class.__name__ in rot_class.__name__
 
     # Check that the base class is in fact the specified class
-    assert type(rot_frame.base) == base_class
+    assert isinstance(rot_frame.base, base_class)
```

Comment:
Need to see if this is ok or if I need to revert it and add the NOQA

Reply:
I'd prefer if this change and the two similar changes in this file were reverted (and this one flagged as NOQA).  This is testing that the frame classes in the RotatedSunFrame machinery are exactly what they are supposed to be, so unexpected subclassing ought to be treated as failures.

Reply:
Makes sense, I will fix this and add the noqa plus the rule.

Reply:
Restored. autopep8 ignores noqas